README.md

```diff
@@ -98,19 +113,21 @@ The `01_cris_clean_up.py` python script contains the functions used to clean and
 
 ### Clinical Measures Data
 
-The `02_ctdb_clean_up.py` python script contains the functions used to clean and convert the spreadsheet downloaded from CTDB to BIDS-standard TSV files as well as their respective data dictionaries converted to BIDS-standard JSON files.
+The `02_ctdb_clean_up.py` python script contains the functions used to clean and convert the spreadsheet downloaded from CTDB, the database used by the study team, to BIDS-standard TSV files as well as their respective data dictionaries converted to BIDS-standard JSON files.
 
-Some of the python functions common to both scripts were defined in `rvoldefinitions.py`
+Some python functions common to both scripts were defined in `rvoldefinitions.py`
 
 ### BIDS-standard MEG Files
 
 Data collected by the NIMH MEG Core was converted to BIDS-standard files using the MNE BIDS package. Associated notebooks: `1_mne_bids_extractor.ipynb` & `2_bids_editor.ipynb`. In addition, event codes and correct timings were generated with `hv_process.py` script, and the fiducial locations from the AFNI HEAD files were generated using the `export_tags.py` script. 
 
 ### BIDS-standard MRI
 
-We used the `heudiconv` tool to convert MRI DICOM files to BIDS-standard files with the associated script: `hv_protocol_heuristic.py`. A modified workflow of `pydeface` was used to deface structural scans with the associated python script: `hv_protocol_pydeface_workflow.py`
+We used the [dcm2bids](https://github.com/UNFmontreal/Dcm2Bids) tool to convert MRI DICOM files to BIDS-standard files. To preserve subject privacy, structural MRI scans are defaced using To preserve subject privacy, structural MRI scans are defaced using [AFNI Refacer version 2.31](https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/tutorials/refacer/refacer_run.html) and defaced scans were checked for quality using [VisualQC v0.6](https://github.com/raamana/visualqc).
```

Comment:
Please include the Dcm2Bids version and the dcm2niix version, since Dcm2Bids is just a wrapper for dcm2niix.